.env

```diff
@@ -0,0 +1 @@
+POSTGRES_PASSWORD=postgrespassword
```

Comment:
As a general rule, you should not be including passwords in files that are committed to version control.  Instead, document that that environment variable needs to be defined, and require it to be defined prior to running compose.

Reply:
I think what I would recommend is renaming this to env.sample, then add a note in the README.md file that the env.sample file should be renamed to .env, and edited as appropriate.  Make sure that .env is added to the .gitignore file so that it's not committed to version control again.